implementation/src/main/java/io/smallrye/jwt/auth/principal/KeyLocationResolver.java

```diff
@@ -147,25 +148,23 @@ PublicKey getHttpsJwk(String kid) {
                     || now > lastForcedRefreshTime + authContextInfo.getForcedJwksRefreshInterval() * 60 * 1000) {
                 lastForcedRefreshTime = now;
                 try {
-                    LOGGER.debug("JWK with a matching 'kid' is not available, refreshing HTTPS JWK Set");
+                    PrincipalLogging.log.kidIsNotAvailableRefreshingJWKSet();
                     httpsJwks.refresh();
                 } catch (JoseException | IOException e) {
-                    LOGGER.debug("Failed to refresh HTTPS JWK Set", e);
+                    PrincipalLogging.log.failedToRefreshJWKSet(e);
                     return null;
                 }
             } else {
-                LOGGER.debugf("JWK with a matching 'kid' is not available"
-                        + " but HTTPS JWK Set has been refreshed less than %d minutes ago"
-                        + authContextInfo.getForcedJwksRefreshInterval());
-                LOGGER.debugf("Trying to create a key from the HTTPS JWK Set one more time");
+                PrincipalLogging.log.matchingKidIsNotAvailableButJWTSRefreshed(authContextInfo.getForcedJwksRefreshInterval());
+                PrincipalLogging.log.tryCreateKeyFromJWKSOneMoreTime();
```

Comment:
Can that be collapsed into a single call, please add `, trying to create a key from the HTTPS JWK Set one more time` to the 1st message

Reply:
Done!